Makefile

```
@@ -215,7 +215,7 @@ $(BASE_BUILD_TARGETS):
 				$(CURDIR)/base
 
 push-base-%: TARGET = $(word 3,$(subst -, ,$@))
-push-base-%: IMAGE_TAG = base-$(word 3,$(subst -, ,$@))-$(KERNEL_FLAVOR)-$(DRIVER_BRANCH)
+push-base-%: IMAGE_TAG = base-$(word 3,$(subst -, ,$@))-$(KERNEL_FLAVOR)-$(DRIVER_BRANCH)-$(LTS_KERNEL)
```

nit: I would prefer a tag in this format jammy-5.15-generic-550
```suggestion
push-base-%: IMAGE_TAG = base-$(word 3,$(subst -, ,$@))-$(LTS_KERNEL)-$(KERNEL_FLAVOR)-$(DRIVER_BRANCH)
```